packages/test/test-utils/src/testContainerRuntimeFactory.ts

```diff
@@ -49,7 +49,7 @@ export const createTestContainerRuntimeFactory = (containerRuntimeCtor: typeof C
             const builder = new RuntimeRequestHandlerBuilder();
             builder.pushHandler(
                 defaultRouteRequestHandler("default"),
-                innerRequestHandler);
+                rootDataStoreRequestHandler);
```

Comment:
Can we just remove `innerRequestHandler` and not add `rootDataStoreRequestHandler`?  It would be good to have the test container runtime factory match the `ContainerRuntimeFactoryWithDefaultDataStore` as closely as possible by default.